invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/utils.js

```diff
@@ -56,12 +58,12 @@ export function SearchItemCreators({ creators }) {
         icon = "/static/images/orcid.svg";
         break;
       case "ror":
-        link = "https://ror.org/"+identifier;
+        link = "https://ror.org/" + identifier;
```

Comment:
As a personal preference I use string templating thanks to previous eslint rules. See [this](https://eslint.org/docs/latest/rules/prefer-template) 

```suggestion
        link = `https://ror.org/${identifier}`;
```